### PR TITLE
enh(release): Add Centreon MAP 20.10.5 RN

### PR DESCRIPTION
**Pull request by lpinsivy**
## Description

Please include a short resume of the changes and what is the purpose of the PR.

Any relevant information should be added to help reviewers.

## Target serie

- [ ] 20.04.x
- [x] 20.10.x
- [ ] 21.04.x
- [ ] 21.10.x (master)

### CODE REVIEW

**Comment by chgautier** [comment]
Please translate the French version

File: fr/releases/centreon-commercial-extensions.md [hunk 1]
```diff
@@ -19,6 +19,16 @@ commerciales, veuillez contacter le support.
 
 ## Centreon MAP
 
+### 20.10.5
+
+#### Bugfixes
```
Comment (chgautier) [inline fr/releases/centreon-commercial-extensions.md:24]:
```suggestion
#### Correctifs
```

File: fr/releases/centreon-commercial-extensions.md [hunk 2]
```diff
@@ -19,6 +19,16 @@ commerciales, veuillez contacter le support.
...
+
+ - [Web-Client] Fixed Graph scale in web interface
```
Comment (chgautier) [inline fr/releases/centreon-commercial-extensions.md:26]: French translation please

File: fr/releases/centreon-commercial-extensions.md [hunk 3]
```diff
@@ -19,6 +19,16 @@ commerciales, veuillez contacter le support.
...
+ - [Web-Client] Fixed "Home" icon in geoviews
```
Comment (chgautier) [inline fr/releases/centreon-commercial-extensions.md:27]: French translation please

File: fr/releases/centreon-commercial-extensions.md [hunk 4]
```diff
@@ -19,6 +19,16 @@ commerciales, veuillez contacter le support.
...
+ - [Web-Client] Fixed Infrastructure View link (BA)
```
Comment (chgautier) [inline fr/releases/centreon-commercial-extensions.md:28]: French translation please

File: fr/releases/centreon-commercial-extensions.md [hunk 5]
```diff
@@ -19,6 +19,16 @@ commerciales, veuillez contacter le support.
...
+ - [Web-Client] Fixed Sorting of maps by alphabetic order
```
Comment (chgautier) [inline fr/releases/centreon-commercial-extensions.md:29]: French translation please

File: fr/releases/centreon-commercial-extensions.md [hunk 6]
```diff
@@ -19,6 +19,16 @@ commerciales, veuillez contacter le support.
...
+ - [Web-Client] Fixed Links to hosts/services
```
Comment (chgautier) [inline fr/releases/centreon-commercial-extensions.md:30]: French translation please

**Review by chgautier — CHANGES_REQUESTED** [review]

File: fr/releases/centreon-commercial-extensions.md (hunk 2 above)
Comment (nmagniezCentreon) [reply fr/releases/centreon-commercial-extensions.md:26]: No translation anymore in RN

**Review by chgautier — APPROVED** [review]

File: fr/releases/centreon-commercial-extensions.md [hunk 7]
```diff
@@ -19,6 +19,16 @@ commerciales, veuillez contacter le support.
 
 ## Centreon MAP
 
+### 20.10.5
+
```
Comment (tuntoja) [inline fr/releases/centreon-commercial-extensions.md:23]:
```suggestion

`21 septembre 2021`

```

**Pull request merged**